apm-agent-benchmarks/src/main/java/co/elastic/apm/agent/benchmark/ProcessOtelBenchmarkResults.java

```diff
@@ -122,7 +122,7 @@ private void writeBulkFile(String resultFilePath) throws IOException {
         final File file = new File(resultFilePath);
         final FileWriter fileWriter = new FileWriter(file);
         for (JsonNode benchmark : bechmarkResultJson) {
-            fileWriter.append("{ \"index\" : { \"_index\" : \"microbenchmarks\" } }\n");
+            fileWriter.append("{ \"index\" : { \"_index\" : \"otel-microbenchmarks\" } }\n");
```

Comment:
This should help with using a specific index for this kind of overhead-benchmark. If the name of the index changes then it requires also changes in the elasticsearch role - so it can write to that particular new index.